reactor-netty-http/src/main/java/reactor/netty/http/client/MicrometerHttpClientMetricsRecorder.java

```diff
@@ -34,12 +34,16 @@
  * @author Violeta Georgieva
  * @since 0.9
  */
-final class MicrometerHttpClientMetricsRecorder extends MicrometerHttpMetricsRecorder implements HttpClientMetricsRecorder {
+public class MicrometerHttpClientMetricsRecorder extends MicrometerHttpMetricsRecorder implements HttpClientMetricsRecorder {
```

Comment:
Let's keep the class final and make it just public with the corresponding constructor. Wdyt?

Reply:
I think it's more practical to keep it public without `final`. The reason is the same for why we created a class `MicrometerHttpClientMetricsRecorder`, that's for abstraction.
If someday the implementation of `MicrometerHttpClientMetricsRecorder` changes, users don't need to know it because they can extend `MicrometerHttpClientMetricsRecorder`.